pages/about.js

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import Container from '@mui/material/Container';
+import Typography from '@mui/material/Typography';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import ProTip from '../src/ProTip';
+import Link from '../src/Link';
+import Copyright from '../src/Copyright';
+
+export default function About() {
+  return (
+    <Container maxWidth="sm">
+      <Box sx={{ my: 4 }}>
+        <Typography variant="h4" component="h1" gutterBottom>
+          Next.js example
+        </Typography>
```

Comment:
Maybe we should change this to lorem or site related content

Reply:
about component is for testing only. I think @Erm0es already has a AboutUs component which is to be used.

Reply:
this isnt needed.

Reply:
Oh yeah, thought I deleted it! Thanks!

Reply:
Fixed.